cmd/main.go

```diff
@@ -355,3 +361,21 @@ func autoMigrationWithLeader(migrationLeader leader.ElectorInterface, db *gorm.D
 		return err
 	})
 }
+
+func uploadBaseISOWithLeader(uploadLeader leader.ElectorInterface, objectHandler s3wrapper.API, generator generator.ISOInstallConfigGenerator, log logrus.FieldLogger) error {
```

Comment:
How do update the image? currently we use the always the same name no?

Reply:
Updating isn't really related to this commit, but right now in s3wrapper/client.go we have:
`const BaseObjectName = "livecd-46.82.202009222340-0.iso"`
So we just change that accordingly.  The plan is later to add the image name to the configmap that describes the OCP versions and get it from there.